internal/cloud/realm/access_list.go

```diff
@@ -26,6 +32,29 @@ type allowedIPCreatePayload struct {
 	UseCurrent bool   `json:"use_current,omitempty"`
 }
 
+func (c *client) AllowedIPs(groupID, appID string) ([]AllowedIP, error) {
+	res, resErr := c.do(
+		http.MethodGet,
+		fmt.Sprintf(allowedIPsPathPattern, groupID, appID),
+		api.RequestOptions{},
+	)
+	if resErr != nil {
+		return nil, resErr
+	}
+
+	if res.StatusCode != http.StatusOK {
+		return nil, api.ErrUnexpectedStatusCode{"get allowed ips and/or CIDR blocks", res.StatusCode}
```

Comment:
[meganit] considering the context of this client call and that this error condition should never hit in the first place, lets just go with:
```suggestion
		return nil, api.ErrUnexpectedStatusCode{"get allowed ips", res.StatusCode}
```